opm/simulators/wells/BlackoilWellModelConstraints.cpp

```diff
@@ -481,32 +481,31 @@ actionOnBrokenConstraints(const Group& group,
         return true;
     }
 
+    const auto action = [&group_limit_action](Group::ProductionCMode control) {
+        switch (control) {
+            case Group::ProductionCMode::ORAT: return group_limit_action.allRates;
+            case Group::ProductionCMode::WRAT: return group_limit_action.water;
+            case Group::ProductionCMode::GRAT: return group_limit_action.gas;
+            case Group::ProductionCMode::LRAT: return group_limit_action.liquid;
+            default: return group_limit_action.allRates;
```

Comment:
There is only possible actions for the limits of oil, water, gas and liquid. Others can only have `RATE` action. 

`FLD` is something different. 

Ideally, default should be `RATE`, while `FLD` either be interpreted by parser or handled here.

Reply:
Amended. FLD will be handled in https://github.com/OPM/opm-common/pull/4635